trunk/SUAVE/Analyses/Aerodynamics/Supersonic_Zero.py

```diff
@@ -68,6 +68,19 @@ def __defaults__(self):
         settings.spoiler_drag_increment             = 0.00 
         settings.oswald_efficiency_factor           = None
         settings.maximum_lift_coefficient           = np.inf 
+        settings.begin_drag_rise_mach_number        = 0.95
+        settings.end_drag_rise_mach_number          = 1.2
+        settings.transonic_drag_multiplier          = 1.25    
```

Comment:
where is the 1.25 from?

Reply:
Information has been added to the source.